.size-snapshot.json

```diff
@@ -1,21 +1,21 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 372617,
```

Comment:
Main code change i can see is our moving from `throw new Error` to `invariant` in an attempt to reduce kb's in tree shaking. However, it looks like our umd bundle went up. Any thoughts @TrySound ?

Reply:
Don't know to be honest.

@lukastaegert what was changed in the latest versions?

Reply:
I updated the snapshot on master so we are sure we are looking at the correct thing

Reply:
I had a look at the build and it is not creating multiple copies of invariant..

Reply:
I think it might be a transitive dependency change..

Reply:
Okay, i reversed the invariant usages changes and i can confirm that the size changes did not come from there